core/src/main/java/org/apache/calcite/sql/validate/SqlValidatorImpl.java

```diff
@@ -6888,6 +6891,31 @@ static class ExtendedExpander extends Expander {
 
       return super.visit(literal);
     }
+
+    /**
+     * Returns whether a given node contains a {@link SqlIdentifier}.
+     *
+     * @param parent a SqlNode
+     * @param children a SqlIdentifier
+     */
+    private boolean containIdentifier(SqlNode parent, SqlIdentifier children) {
```

Comment:
`containIdentifier` -> `containsIdentifier`?

Reply:
Good catch, thanks for your review.